src/components/ha-date-range-picker.ts

```diff
@@ -261,51 +219,44 @@ export class HaDateRangePicker extends LitElement {
       >
         <div slot="input" class="date-range-inputs" @click=${this._handleClick}>
           ${!this.minimal
-            ? html`<ha-svg-icon .path=${mdiCalendar}></ha-svg-icon>
-                <ha-icon-button-prev
-                  .label=${this.hass.localize("ui.common.previous")}
-                  class="prev"
-                  @click=${this._handlePrev}
-                >
-                </ha-icon-button-prev>
-                <ha-textfield
-                  .value=${this.timePicker
-                    ? formatDateTime(
+            ? html`<ha-textarea
+                  mobile2multiline
+                  .value=${(isThisYear(this.startDate)
+                    ? formatShortDateTime(
                         this.startDate,
                         this.hass.locale,
                         this.hass.config
                       )
-                    : formatDate(
+                    : formatShortDateTimeWithYear(
                         this.startDate,
                         this.hass.locale,
                         this.hass.config
-                      )}
-                  .label=${this.hass.localize(
-                    "ui.components.date-range-picker.start_date"
-                  )}
-                  .disabled=${this.disabled}
-                  @click=${this._handleInputClick}
-                  readonly
-                ></ha-textfield>
-                <ha-textfield
-                  .value=${this.timePicker
-                    ? formatDateTime(
+                      )) +
+                  " - \n" +
+                  (isThisYear(this.endDate)
+                    ? formatShortDateTime(
                         this.endDate,
                         this.hass.locale,
                         this.hass.config
                       )
-                    : formatDate(
+                    : formatShortDateTimeWithYear(
                         this.endDate,
                         this.hass.locale,
                         this.hass.config
-                      )}
+                      ))}
                   .label=${this.hass.localize(
-                    "ui.components.date-range-picker.end_date"
+                    "ui.components.selectors.selector.types.datetime"
```

Comment:
Don't use the translation key of another selector, you can reference its value though